src/supervisor/types.ts

```diff
@@ -1,3 +1,4 @@
+import * as http from 'http';
```

Comment:
What do you think of importing just the thing we need, e.g. `import { IncomingMessage } from 'http'`?